lib/galaxy/webapps/galaxy/api/display_applications.py

```diff
@@ -13,6 +13,7 @@
 class DisplayApplicationsController(BaseAPIController):
 
     @expose_api
+    @require_admin
```

Comment:
Out of curiosity, why do we require admin for these?  Is there anything privileged in the response, or is it just historical?

Seems like this is info we'd expose just like the /api/tools index.

(obviously the reload/post below should require admin)

Reply:
Historical. The legacy endpoint which previously produced this display was in the admin controller. I preserved the same level of access requirements.